ts/modules/parallel-coordinates.src.ts

```diff
@@ -174,6 +163,190 @@ H.setOptions({
+                axis.parallel = new ParallelAxisAdditions(axis as ParallelAxis);
+            }
+        });
+
+        // Update default options with predefined for a parallel coords.
+        addEvent(AxisClass, 'afterSetOptions', function (
+            e: {
+                userOptions: Highcharts.XAxisOptions;
+            }
+        ): void {
+            var axis = this,
+                chart = axis.chart,
+                parallel = axis.parallel as ParallelAxisAdditions,
+                axisPosition = ['left', 'width', 'height', 'top'];
+
+            if (chart.hasParallelCoordinates) {
+                if (chart.inverted) {
+                    axisPosition = axisPosition.reverse();
+                }
+
+                if (axis.isXAxis) {
+                    axis.options = merge(
+                        axis.options,
+                        defaultXAxisOptions,
+                        e.userOptions
+                    );
+                } else {
+                    axis.options = merge(
+                        axis.options,
+                        (axis.chart.options.chart as any).parallelAxes,
+                        e.userOptions
+                    );
+                    parallel.position = pick(
+                        parallel.position,
+                        chart.yAxis.length
+                    );
+                    parallel.setPosition(axisPosition, axis.options);
+                }
+            }
+        });
+
+
+        // Each axis should gather extremes from points on a particular position
+        // in series.data. Not like the default one, which gathers extremes from
+        // all series bind to this axis. Consider using series.points instead of
+        // series.yData.
+        addEvent(AxisClass, 'getSeriesExtremes', function (e: Event): void {
+            const axis = this;
+            const parallel = axis.parallel;
+
+            if (!parallel) {
+                return;
+            }
+
+            if (this.chart && this.chart.hasParallelCoordinates && !this.isXAxis) {
+                var index = parallel.position,
+                    currentPoints: Array<Highcharts.Point> = [];
+
+                this.series.forEach(function (series: Highcharts.Series): void {
+                    if (
+                        series.visible &&
+                        defined((series.yData as any)[index as any])
+                    ) {
+                        // We need to use push() beacause of null points
+                        currentPoints.push((series.yData as any)[index as any]);
+                    }
+                });
+                this.dataMin = arrayMin(currentPoints);
+                this.dataMax = arrayMax(currentPoints);
+
+                e.preventDefault();
+            }
+        });
+
+        /* eslint-enable no-invalid-this */
+
+    }
+}
+
+interface ParallelAxis extends Axis {
+    chart: Highcharts.ParallelChart;
+    parallel: ParallelAxisAdditions;
```

Comment:
I think this one should be called `parallelCoordinates` to avoid any future confusion.